internal/apiclient/eventstreams.go

```diff
@@ -31,7 +31,7 @@ func (c *fftmClient) GetEventStreams(ctx context.Context) ([]apitypes.EventStrea
 		SetResult(&eventStreams).
 		Get("eventstreams")
 	if !resp.IsSuccess() {
-		return nil, fmt.Errorf(string(resp.Body()))
+		return nil, fmt.Errorf("%s", string(resp.Body()))
```

Comment:
This doesn't make sense to me.
Did you mean to address the lint issue that comes up with `fmt.Errorf` with a static string? If so, you should move to the `errors.New(text string)` function

Reply:
I think this was done automatically by the golangci-lint?

Reply:
It should be raised tehre - but the recommendation is to use `errors.New()` I believe. I've never used some kind of "auto-fix" if that's what you're talking about, so I don't know what that would do.

My feedback remains

Reply:
I'll switch to errors.New(). It was fixed using code pilot.